eng/scripts/Armstrong-Validation.ps1

```diff
@@ -0,0 +1,151 @@
+[CmdletBinding()]
+param (
+  [Parameter(Position = 0)]
+  [string] $BaseCommitish = "HEAD^",
+  [Parameter(Position = 1)]
+  [string] $TargetCommitish = "HEAD"
+)
+Set-StrictMode -Version 3
+
+. $PSScriptRoot/ChangedFiles-Functions.ps1
+. $PSScriptRoot/Logging-Functions.ps1
+
+function Get-Suppression {
+  param (
+    [string]$fileInSpecFolder
+  )
+
+  # -NoEnumerate to prevent single-element arrays from being collapsed to a single object
+  # -AsHashtable is closer to raw JSON than PSCustomObject
+  $suppressions = npx get-suppressions ArmstrongValidation $fileInSpecFolder | ConvertFrom-Json -NoEnumerate -AsHashtable
```

Comment:
@ms-zhenhua: I updated your PR to use the new `get-suppressions` shared component added in #28927.

You may also want to add parameter `$CheckAllUnder` to your script for easier testing, like this:

https://github.com/Azure/azure-rest-api-specs/pull/28927/files#diff-3c37fefe028ad9bd154b76ea3371f8db2d7a7609dafa323e92d8461880e808b2R9